Pickers/SWActionSheet.m

```diff
@@ -260,7 +256,8 @@ - (BOOL)prefersStatusBarHidden
 
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceiveTouch:(UITouch *)touch
 {
-    if (touch.view.tag == sheetBackgroundTag) {
+    CGPoint location = [touch locationInView:_actionSheet];
+    if (CGRectContainsPoint(_actionSheet.bgView.frame, location)) {
         return NO;
     }
     return YES;
```

Comment:
```suggestion
    return !(CGRectContainsPoint(_actionSheet.bgView.frame, location);
```

Reply:
Sure i'll make that change of course :) Only reason for the current style is because it makes it easier to add break points for debugging specific cases.